SECURITY-INSIGHTS.yml

```diff
@@ -0,0 +1,55 @@
+header:
+  schema-version: 1.0.0
+  expiration-date: '2024-12-16'
+  project-url: 
+project-lifecycle:
```

Comment:
Curious if we are missing a `release-process` reference like shown here: https://github.com/kubernetes-sigs/kueue/pull/1469/files#diff-000b1100271690c8bb5643a7f806f131a81fdb22cf27af740f5d645e985e9811R19

Reply:
we don't have `NEW_RELEASE.md` file under https://github.com/kubernetes-sigs/karpenter/tree/main/.github/ISSUE_TEMPLATE